scripts/release/increment_version_tag.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+# shellcheck source=scripts/lib.sh
+source "$(dirname "$(dirname "${BASH_SOURCE[0]}")")/lib.sh"
+cdroot
+
+usage() {
+	cat <<EOH
+Usage: ./increment_version_tag.sh [--dry-run] [--ref <ref>] <--major | --minor | --patch>
```

Comment:
How about:

```suggestion
Usage: ./tag_version.sh [--dry-run] [--ref <ref>] <--major | --minor | --patch>
```

and rename the file too

Reply:
I think the former is clearer in what it does, but I don't have a strong opinion on this.